src/main.js

```diff
@@ -27,6 +28,24 @@ Vue.use(VueSocketIO, socket);
 // Set up Vue Router
 Vue.use(VueRouter);
 
+// Set up PortalGun (connection to native app)
+PortalService.listen();
+
+const handlePortalData = data => {
+  // TODO: don't route immediately if push is received while app is open.
+  // can detect with `if (data._isPush && data._original?.additionalData?.foreground)`
+  // and show own UI for notification
+  if (data.path) {
+    router.push(data.path);
```

Comment:
@austinhallock is this supposed to route the Vue app to the path variable passed here? https://github.com/UPchieve/server/pull/291/files#diff-0588366fd7fe31e04d97ceb24dc23788R47

Reply:
yes, when app is opened via push notification